src/main/resources/google_checks.xml

```diff
@@ -285,6 +285,13 @@
       <property name="lineWrappingIndentation" value="4"/>
       <property name="arrayInitIndent" value="2"/>
     </module>
+    <!-- Suppression for block code until we find a way to detect them properly
+         https://github.com/checkstyle/checkstyle/issues/15769 -->
```

Comment:
Please put until word before link, link and such word should be on same line.
This is how we detect closed issues.

Reply:
done